omnichain/staking/contracts/Staking.sol

```diff
@@ -9,15 +9,17 @@ import "@zetachain/toolkit/contracts/BytesHelperLib.sol";
 contract Staking is ERC20, zContract {
     error SenderNotSystemContract();
     error WrongChain();
-    error NotAuthorizedToClaim();
+    error UnknownAction();
```

Comment:
I think you can set attributes to error, `UnknownAction(uint action)`. In practice this allows to have a more explicit error, this might be a good thing to show in the tutorial

Reply:
d32bb85